tests/integration/test_script_runner.py

```diff
@@ -23,15 +23,10 @@ class BackendDependencies(enum.Enum):
     SNOWFLAKE = "SNOWFLAKE"
 
 
-docs_test_matrix = [
-    {
-        "user_flow_script": "tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs_yaml_configs.py",
-        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
-    },
-    {
-        "user_flow_script": "tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_bigquery_yaml_configs.py",
-        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
-    },
+# to be populated by the smaller lists below
+docs_test_matrix = []
+
```

Comment:
Could we add a blurb here about what this matrix is doing and what its contents look like?

Additionally, some information around the schema of the dictionary objects would be a huge help for newer folks to the team (what's required, what's optional, etc). I think a `TypedDict` would be ideal for this but it's 3.8-specific so we can't use it.

Perhaps a `namedtuple` could be handy to provide a lightweight schema but I don't think it's entirely necessary for a test runner like this.

Reply:
Thank you very much for this suggestion @cdkini, as we discussed I'll be including this change in subsequent PR